src/main/webapp/authorization.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * The ID of the login button
+ * @type {String}
+ */
+const LOGIN_BTN = 'login-btn';
+
+// GCP Generated Github 'OAuth' configuration
+let config = {
+  apiKey: 'AIzaSyAR88Giah8cCEAvT_zDSIREWvgIIAeS8yY',
+  authDomain: 'step2020-279820.firebaseapp.com',
+};
+firebase.initializeApp(config);
+
+/**
+ * Function called when clicking the Login/Logout button.
+ */
+function toggleSignIn() {
+  // if not logged in
+  if (!firebase.auth().currentUser) {
+    let provider = new firebase.auth.GithubAuthProvider();
+    // sign in
+    firebase.auth().signInWithPopup(provider).then(function(result) {
+      // This gives you a GitHub Access Token. 
+      // You can use it to access the GitHub API.
+      let token = result.credential.accessToken;
+      // The signed-in user info.
+      let user = result.user;
```

Comment:
Same nit as above: 
```suggestion
      // This gives you a GitHub Access Token. 
      // You can use it to access the GitHub API.
      const token = result.credential.accessToken;
      // The signed-in user info.
      const user = result.user;
```

Also, this data isn't currently going anywhere / being used. In the future, we should make this into a module so the user data can be accessed around the application.